### DISCUSSION TITLE
[OSPP] 想问一些社区OSPP的问题

### DISCUSSION THREAD

**Discussion by sususama** [sofastack/sofa-rpc#1324] [Q&A] [answered]
Hi,
我想参加社区今年的OSPP，不知道现阶段应该做些什么。
社区的Issues现在看起来都有人在回答或者解决。

**Comment by OrezzerO** [comment]
@EvenLjj Any suggestions?

**Comment by EvenLjj** [comment, marked as answer]
@sususama 现阶段可以先多了解下SOFASTACK的[产品](https://www.sofastack.tech/projects/)。OSPP 目前还处于出题阶段，可以先关注[开源之夏官网](https://summer-ospp.ac.cn/)。题目确定之后，也会把任务issue列出来，请关注后续社区 issue 列表。
